bitnami/redis/templates/sentinel/statefulset.yaml

```
@@ -756,6 +756,7 @@ spec:
       kind: PersistentVolumeClaim
       metadata:
         name: redis-data
+        {{- $claimLabels := include "common.tplvalues.merge" ( dict "values" ( list .Values.replica.persistence.labels .Values.commonLabels ) "context" . ) }}
         labels: {{- include "common.labels.matchLabels" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 10 }}
```

I think this has been missing. [Eg](https://github.com/bitnami/charts/blob/main/bitnami/rabbitmq/templates/statefulset.yaml#L468)
```suggestion
        labels: {{- include "common.labels.matchLabels" ( dict "customLabels" $claimLabels "context" $ ) | nindent 10 }}
```